runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentView.java

```diff
@@ -102,4 +102,29 @@ protected void onFocusChanged(boolean gainFocus, int direction, Rect previouslyF
         mXWalkView.onFocusChangedDelegate(gainFocus, direction, previouslyFocusedRect);
         super.onFocusChanged(gainFocus, direction, previouslyFocusedRect);
     }
+
+    @Override
+    public int computeHorizontalScrollOffset() {
+        return super.computeHorizontalScrollOffset();
+    }
+
+    @Override
+    public int computeHorizontalScrollRange() {
+        return super.computeHorizontalScrollRange();
+    }
+
+    @Override
+    public int computeVerticalScrollExtent() {
+        return super.computeVerticalScrollExtent();
+    }
+
+    @Override
+    public int computeVerticalScrollOffset() {
+        return super.computeVerticalScrollOffset();
+    }
+
+    @Override
+    public int computeVerticalScrollRange() {
+        return super.computeVerticalScrollRange();
+    }
```

Comment:
If there's no special transition, no need to implement these methods in XWalkContentView.